save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt

```diff
@@ -434,6 +455,36 @@ class VulnerabilityService(
         .orNotFound { "Could not find date for ${dateDto.vulnerabilityName} of type ${dateDto.type} (${dateDto.date})." }
         .let { vulnerabilityDateRepository.delete(it) }
 
+    /**
+     * @param vulnerabilityName [Vulnerability.name]
+     * @param tag tag to add
+     * @return updated [Vulnerability]
+     */
+    @Transactional
+    fun addTag(vulnerabilityName: String, tag: String): Vulnerability {
```

Comment:
maybe create a dedicated table for tags and link with vulnerability by foreign key (many to many)?
It does allow to search all vulnerability by tag without manipulating with strings

Reply:
Adding a new table (or even two of them) seem to solve several problems, but it also brings more complexity. Will do it, but probably in next PR if it is ok for you - this PR already is quite big

Reply:
> Reference in new issue
> 
> ### Reference in new issue
> Repository

only if you really will do it now in next PR :)